### PR TITLE
[TECHNICAL SUPPORT] LPS-102708 Window horizontal scrollbar will change upon hovering the mouse over "flat" Navigation Menu items

### PR DESCRIPTION
**Pull request by NemethNorbert**
Hello @antonio-ortega,

[LPS-102708](https://issues.liferay.com/browse/LPS-102708),
Please review it:)

Thanks,

### CODE REVIEW

**Comment by NemethNorbert** [comment]
ci:test:sf

**Comment by NemethNorbert** [comment]
ci:test:sf

**Comment by NemethNorbert** [comment]
ci:test:relevant

**Comment by NemethNorbert** [comment]
ci:test:sf

**Comment by NemethNorbert** [comment]
ci:test:relevant

**Comment by NemethNorbert** [comment]
Hey @antonio-ortega,

I simplified the solution

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/navigation_interaction.js [hunk 1]
```diff
@@ -333,13 +333,15 @@ AUI.add(
...
 						function(event) {
-							var showMenu = event.type == 'showNavigationMenu';
-
 							var menu = event.menu;
 
 							if (menu) {
 								instance._lastShownMenu = null;
 
+								var showMenu =
+									event.type == 'showNavigationMenu' &&
```
Comment (jbalsas) [inline modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/navigation_interaction.js]: This should be `===`. We always use strict comparison.

**Comment by antonio-ortega** [comment]
brianchandotcom#79617

**Pull request closed without merging**